liferay-echo/liferay-portal
# LPD-23888 Deprecate

## Epic

https://liferay.atlassian.net/browse/LPD-11377

### Feature Flag
`feature.flag.LPD-11377=true`

### Description
This PR select which item display when mapping repeatable fields. Mapping was possible but always showed the first item. Currently only work for tags and categories (which now are considered repeatable). The rest of repeatable fields will be implemented in the following prs

### How to test it

- Add some tags and categories
- Add a web content with those categories and asset
- Create a content page
- Add a heading
- Map the web content and choose asset or tags
- A new selector should appear allowing user to select first, last or iteration number
